src/qt/peertablemodel.h

```diff
@@ -74,7 +74,33 @@ public Q_SLOTS:
 
 private:
     interfaces::Node& m_node;
-    const QStringList columns{tr("Peer Id"), tr("Address"), tr("Type"), tr("Network"), tr("Ping"), tr("Sent"), tr("Received"), tr("User Agent")};
+    const QStringList columns{
+        /*: Title of Peers Table column which contains a
+            unique number used to identify a connection. */
+        tr("Peer"),
+        /*: Title of Peers Table column which contains the
+            IP/Onion/I2P address we used to connect to the peer. */
+        tr("Address"),
+        /*: Title of Peers Table column which contains a value for
+            the type of connection we have established with the peer.
```

Comment:
"a value for the type" -- sounds unusual for me.

Reply:
agree that "we have established" sounds like outbound

"which describes the type of peer connection" ?